docs/source/cuda/overview.rst

```diff
@@ -2,6 +2,55 @@
 Overview
 ========
 
+.. cuda-deprecated::
+
+.. _cuda-deprecation-status:
+
+Built-in CUDA target deprecation and maintenance status
+=======================================================
+
+The CUDA target is now maintained in a separate package, `numba-cuda
+<https://nvidia.github.io/numba-cuda>`_. This enables improvements in the
+development of the CUDA target:
+
+* The time and effort required to incorporate new features and bug fixes into
+  the CUDA target is decreased - its development cycle is now decoupled from the
+  Numba development process, so "upstream" reviews from Numba maintainers and
+  test runs on the Anaconda internal CI systems are no longer required as part
+  of the standard process for CUDA target pull requests. This lightening of
+  process enables development to proceed at an increased cadence.
+* Similarly, releases of the CUDA target can be made independently of Numba
+  releases, at a more frequent pace.
+* Numba is sufficiently mature as a compiler platform to support out-of-tree
+  targets. The CUDA target, whilst maintained upstream, had been migrated to
+  use externally-facing APIs for target implementation. The continued
```

Comment:
```suggestion
  use externally-facing APIs for the target implementation. The continued
```

Reply:
I think this change changes the meaning to something I don't mean - the externally facing APIs are for implementing targets, not "the targets". Instead I think this keeps the original meaning:

```suggestion
  use the externally-facing APIs for target implementation. The continued
```

Reply:
I agree with your comment, thanks for checking.